pkg/email/send_email.go

```diff
@@ -0,0 +1,44 @@
+package email
+
+import (
+	"github.com/treeverse/lakefs/pkg/email/params"
+	gomail "gopkg.in/gomail.v2"
+)
+
+func NewEmail(e params.Email, err error) (*params.Email, error) {
```

Comment:
What's the purpose of this function?

Reply:
The idea is the basic email params (i.e. SMTP, port, password, etc) should be initialized on `run`, this is done using this function as it takes in the config email struct and creates a new one that can be passed on. Please see the above comment regarding the initilization, and that some of the wiring will be done in a different PR as it is to be determined where it should reside.

Reply:
I'm a bit confused on which functions will trigger this one if we have the `SendEmail` function.
Also, it looks like this function is only storing the email struct information, within a new but same struct and returns it.

We sometimes use "NewStruct" functions in order to simulate object creation (from OOP) using the receiver function, but this is not the case here...
Maybe I miss some context or the rest of the functions to understand it better, let's discuss this tomorrow during our PR review.